scripts/change_url

```diff
@@ -89,48 +105,55 @@ fi
 # SPECIFIC MODIFICATIONS
 #=================================================
 # SETUP LSTU
 #=================================================
-ynh_print_info "Updating lstu configuration..."
+ynh_print_info --message="Updating lstu configuration..."
 
 domain="$new_domain"
 path_url="$new_path"
 
-ynh_backup_if_checksum_is_different "$final_path/lstu.conf"
-cp ../conf/lstu.conf.template "${final_path}/lstu.conf"
-ynh_replace_string "__DOMAIN__" "$domain" "${final_path}/lstu.conf"
-ynh_replace_string "__PATH__" "$path_url" "${final_path}/lstu.conf"
-ynh_replace_string "__PORT__" "$port" "${final_path}/lstu.conf"
-ynh_replace_string "__DB_NAME__" "$db_name"  "${final_path}/lstu.conf"
-ynh_replace_string "__DB_USER__" "$db_user"  "${final_path}/lstu.conf"
-ynh_replace_string "__DB_PWD__" "$db_pwd" "${final_path}/lstu.conf"
-ynh_replace_string "__SELECTED_THEME__" "$theme" "${final_path}/lstu.conf"
-ynh_replace_string "__PASSWORD_HASHED__" "$hashed_password" "${final_path}/lstu.conf"
-
-ynh_replace_string "__SECRET__" "$secret" "${final_path}/lstu.conf"
+config="${final_path}/lstu.conf"
+ynh_backup_if_checksum_is_different --file="$config"
+cp ../conf/lstu.conf.template "$config"
+ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$config"
+ynh_replace_string --match_string="__PATH__" --replace_string="$path_url" --target_file="$config"
+ynh_replace_string --match_string="__PORT__" --replace_string="$port" --target_file="$config"
+ynh_replace_string --match_string="__DB_NAME__" --replace_string="$db_name" --target_file="$config"
+ynh_replace_string --match_string="__DB_USER__" --replace_string="$db_user" --target_file="$config"
+ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file="$config"
+ynh_replace_string --match_string="__SELECTED_THEME__" --replace_string="$theme" --target_file="$config"
+ynh_replace_string --match_string="__PASSWORD_HASHED__" --replace_string="$hashed_password" --target_file="$config"
+ynh_replace_string --match_string="__SECRET__" --replace_string="$secret" --target_file="$config"
 if [ $is_public -eq 0 ];
 then
-	ynh_replace_string "__IS_PUBLIC__" "" "${final_path}/lstu.conf"
+	ynh_replace_string --match_string="__IS_PUBLIC__" --replace_string="" --target_file="$config"
 else
-	ynh_replace_string "__IS_PUBLIC__" "#" "${final_path}/lstu.conf"
+	ynh_replace_string --match_string="__IS_PUBLIC__" --replace_string="#" --target_file="$config"
 fi
-ynh_store_file_checksum "${final_path}/lstu.conf"
+ynh_store_file_checksum --file="$config"
 
 #=================================================
 # GENERIC FINALISATION
+#=================================================
+# START SYSTEMD SERVICE
+#=================================================
+ynh_print_info --message="Starting a systemd service..."
+
+ynh_systemd_action --service_name=$app --action="start" --log_path="systemd" --line_match="Starting hot deployment for Hypnotoad server"
```

Comment:
Tested on Buster, everything work except that line is never found.
Would probably be easier to look for "Server available at"

Reply:
Changed, waiting for package_check results

Reply:
Change made